scripts/deploy/iks/deploy-kfp.sh

```diff
@@ -41,6 +41,9 @@ NEW_PERSISTENCEAGENT_IMAGE="${NEW_PERSISTENCEAGENT_IMAGE:-"${REGISTRY_URL}/${REG
 SCHEDULEDWORKFLOW_IMAGE="${SCHEDULEDWORKFLOW_IMAGE:-"docker.io/aipipeline/scheduledworkflow"}"
 NEW_SCHEDULEDWORKFLOW_IMAGE="${NEW_SCHEDULEDWORKFLOW_IMAGE:-"${REGISTRY_URL}/${REGISTRY_NAMESPACE}/scheduledworkflow:${IMAGE_TAG}"}"
 
+CACHESERVER_IMAGE="${CACHESERVER_IMAGE:-"docker.io/aipipeline/cache-server"}"
+NEW_CACHESERVER_IMAGE_IMAGE="${NEW_CACHESERVER_IMAGE_IMAGE:-"${REGISTRY_URL}/${REGISTRY_NAMESPACE}/cache-server:${IMAGE_TAG}"}"
```

Comment:
Thanks @yhwang, I think we need to do the same for the ml-pipeline-ui image as well